benchmark/benchmain/main.go

```diff
@@ -400,11 +452,28 @@ func setMode(name string) []bool {
 		return []bool{false, true}
 	default:
 		log.Fatalf("Unknown %s setting: %v (want one of: %v)",
-			name, name, strings.Join(allCompressionModes, ", "))
+			name, name, strings.Join(allTraceModes, ", "))
 		return []bool{}
 	}
 }
 
+func setModeCompressor(name string) []string {
+	switch name {
+	case modeNop:
+		return []string{"off", "nop"}
```

Comment:
```suggestion
		return []string{"nop"}
```

Reply:
For the rest of the flags, if an option is given, we only run it. To be consistent with that behavior, please update this.